CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ CHANGELOG
 
 ## HEAD (Unreleased)
 
+- Relax stack name validations for Automation API [#5337](https://github.com/pulumi/pulumi/pull/5337)
```

Comment:
I don't think this is CHANGELOG worthy?

Reply:
technically it's a bugfix for OSS backend users, but happy to remove if you think it shouldn't be here.

Reply:
nah you are right - it's a good bugfix to note - I wonder if it's better to phrase something like "support OSS backends for automation-api"? 

Trying to make changelogs useful... according to an issue that was opened before ;)

We can change it later though - we don't need to gate the PR